packages/docs/src/hooks/use-showcase.ts

```diff
@@ -7,18 +7,21 @@ import { useEffect, useState } from 'react';
 import { ResourceRecord, RegistryDataRecord, CID } from '@dxos/registry-client';
 import { useRegistry } from '@dxos/react-registry-client';
 
+import 'setimmediate';
+import hash from 'string-hash';
 import { SHOWCASE_APPS } from '../data';
 
 // TODO(wittjosiah): Use proto definitions.
 export interface App {
-  hash: Uint8Array;
-  repository?: string;
-  repositoryVersion?: string;
-  license?: string;
-  keywords?: string[];
-  displayName?: string;
-  contentType?: string[];
-  extension?: any;
+  hash: Uint8Array
+  repository?: string
+  repositoryVersion?: string
+  license?: string
+  keywords?: string[]
+  displayName?: string
+  contentType?: string[]
+  extension?: any
+  imageDataUri: string
```

Comment:
Changing this won't actually work now, since we're not using the mock registry this should match the real one because this data will never be there.